src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/AESetup.java

```diff
@@ -505,16 +505,19 @@ protected static void createScaleTable(String tableName, String table[][], Strin
      * @param keyPath
      * @throws SQLException
      */
-    private static void createCMK(String cmkName, String keyStoreName, String keyPath,
+    protected static void createCMK(String connectionString, String cmkName, String keyStoreName, String keyPath,
             String signature) throws SQLException {
         try (SQLServerConnection con = (SQLServerConnection) PrepUtil
-                .getConnection(AETestConnectionString + ";sendTimeAsDateTime=false", AEInfo);
+                .getConnection(connectionString + ";sendTimeAsDateTime=false", AEInfo);
                 SQLServerStatement stmt = (SQLServerStatement) con.createStatement()) {
             String sql = " if not exists (SELECT name from sys.column_master_keys where name='" + cmkName + "')"
                     + " begin" + " CREATE COLUMN MASTER KEY " + cmkName + " WITH (KEY_STORE_PROVIDER_NAME = '"
                     + keyStoreName + "', KEY_PATH = '" + keyPath + "'"
                     + (TestUtils.isAEv2(con) ? ",ENCLAVE_COMPUTATIONS (SIGNATURE = " + signature + ")) end" : ") end");
             stmt.execute(sql);
+        } catch (Exception ex) {
+            // kz debug
+            ex.printStackTrace();
```

Comment:
check for expected error msg and fail otherwise.

Reply:
This is for debugging only. Will remove it when doing clean up.